source/content/terminus/02-install.md

```diff
@@ -15,6 +15,7 @@ audience: [development]
 product: [terminus]
 integration: [--]
 ---
+<Wistia src="mi456ux965" />
```

Comment:
I recommend skipping this video. It has an off-brand Pantheon logo and I think the stock videos here are out of place for the material. I think this substance is better in text only. Or if it is a video, it should be a video of screen capture of Terminus being installed. Not video that shows the same text as is on the page already.